### DISCUSSION TITLE
osxfuse still on 3.11.2 in brew, but shows 4.0.5 available in settings GUI

### DISCUSSION THREAD

**Discussion by maichai** [Homebrew/discussions#689] [Casks] [answered]
osxfuse still on 3.11.2 in brew, but shows 4.0.5 available in settings GUI

**Comment by carlocab** [comment]
You can open a PR to update it. This might help you get started:

https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask

**Comment by Bo98** [comment, marked as answer]
Version 4.0.5 is available under the name `macfuse`. The reason `osxfuse` still exists is because of backwards compatibility with formula in homebrew-core which depend on the old `osxfuse`.
